collect_diag.sh

```diff
@@ -153,6 +153,10 @@ declare -A servers
 for host in $(cat "$HOST_FILE"); do
     debug "Copying collect_node_diag.sh to $host..."
     scp $SSH_OPTS "${LAUNCH_PATH}collect_node_diag.sh" "${host}:~/"
+    if [ "$TYPE" = "coss" ]; then
+        debug "Copying sjk jar to $host..."
+        scp $SSH_OPTS "${LAUNCH_PATH}/libs/sjk-plus.jar" "${host}:~/"
```

Comment:
#nit We don't need the `/` character between `${LAUNCH_PATH}` & `lib/sjk-plus.jar` here as we already have it in here, https://github.com/DataStax-Toolkit/diagnostic-collection/blob/master/collect_diag.sh#L153 as `[[ $0 == */* ]] && LAUNCH_PATH=${0%/*}/ || LAUNCH_PATH=./`